src/Microsoft.Health.Fhir.Shared.Core/Features/Resources/Patch/JsonPatchService.cs

```diff
@@ -74,6 +74,19 @@ private static void Validate(ResourceWrapper currentDoc, WeakETag eTag)
             {
                 throw new PreconditionFailedException(string.Format(Core.Resources.ResourceVersionConflict, eTag.VersionId));
             }
+
+            foreach (var operation in patchDocument.Operations)
+            {
+                if (operation.OperationType == AspNetCore.JsonPatch.Operations.OperationType.Invalid)
+                {
+                    throw new BadRequestException($"{operation.op} is invalid.");
+                }
+
+                if (operation.path == null)
```

Comment:
Should this be?
```suggestion
                if (string.IsNullOrWhiteSpace(operation.path))
```

Reply:
I think empty path is apply to whole resource, so I think it's valid thing.

Also I don't want to write my own validation for operations, it should come from library, so I'll just try to catch ANE during patching instead of this.